core/deployment/src/main/java/io/quarkus/deployment/builditem/AllCharsetsBuildItem.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class AllCharsetsBuildItem extends MultiBuildItem {
```

Comment:
Maybe rename it `NativeEnableAllCharsetsBuildItem`

And maybe add a javadoc comment:
```
Enable the support of all charsets in native mode.
<p>
This is necessary for some extensions, in particular the Microsoft SQL Server extension.
```